.github/workflows/build-yocto.yml

```diff
@@ -63,10 +63,10 @@ jobs:
 
       - name: Kas build
         run: |
-          export DL_DIR=/srv/gh-runners/quic-yocto/downloads
-          export SSTATE_DIR=/srv/gh-runners/quic-yocto/sstate-cache
+          export PERSISTENT_DIR=/srv/gh-runners/quic-yocto
+          export DL_DIR=${PERSISTENT_DIR}/downloads
+          export SSTATE_DIR=${PERSISTENT_DIR}/sstate-cache
+          export KAS_BUILD_DIR=$(realpath $PWD/..)/build
```

Comment:
I think it is actually better to set KAS_WORK_DIR instead (KAS_BUILD_DIR is ${KAS_WORK_DIR}/build), that way it also clones the other repos outside the folder where kas build gets called (which is this repo).

Reply:
ok, I moved it to the outside of the repo `../kas`, one dir node above.